src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        lottery.getRandomBall();
+        lottery.getRandomBall();
+        lottery.getRandomBall();
```

Comment:
Use loop to create several objects, see [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-for-loop-for-creating-several-objects-of-the-same-class). And move the `System.out.println()` here.